printer_mods/Ferdi360/2.4_Bedfan_Backplates/README.md

```diff
@@ -0,0 +1,25 @@
+# Bedfan Backplates for Voron 2.4
+**What is it?**
+Backplates for your Voron 2.4 that block hot air out of your bedfans from blowing onto your ACM panels to (slightly) improve airflow and temps. Also a nice way to showoff Voron branding and the serial of your printer.
+
+I noticed a hotspot on the lower third of the ACM backpanel, which now gets more equally warm. Also the top panel gets warmer sooner, which indicated improved flow for me. Good insulation still is vital to keep the temp in the printer! 
+
+**IMPORTANT NOTICE:**
+If you have standard bed spacers (8 mm), a magnet+PEI sheet and the normal Stealthburner toolhead, you should have 6 mm margin between the gantry and the backplate. 
+You should also make sure you have managed your AB motor cables (and connectors if you added those), so they don't get crushed.
+
+**BOM:**
+- 4x 8mm M3 SHCS (2x for 250mm version)
+- 4x hammerhead t-nut (2x for 250mm version)
+
+**Instructions:**
+The STL's are “blank” plates without serial numbers. I added a Fusion360 file in which you can edit the text in the similar named sketch. Or you can add text in Prusaslicer for example.  
+Print with 3 perimeters and 1% infill. If you do a color change after layer one, you get a nice effect. 
+Stick on a bit of 1mm foam tape (BOM for the back and top panel) to prevent it from resonating against the back panel.
+Mount it with a few (according to size) 8mm M3 SHCS and hammerhead t-nuts.
+
+I have tested on a 350 2.4, i would love to hear if the smaller sizes work well too! 
+
+![CAD](Images/2.4_Backplates.PNG)Images/2.4_Backplates.PNG
+![Photo 1](Images/2.4_Backplates_1.jpg)Images/2.4_Backplates_1.jpg
+![Photo 2](Images/2.4_Backplates_2.jpg)Images/2.4_Backplates_2.jpg
```

Comment:
These filenames could still be removed, but this isn't a required change, just a preference.

```suggestion
![CAD](Images/2.4_Backplates.PNG)
![Photo 1](Images/2.4_Backplates_1.jpg)
![Photo 2](Images/2.4_Backplates_2.jpg)
```